gr-blocks/lib/complex_to_interleaved_char_impl.cc

```diff
@@ -37,13 +41,11 @@ int complex_to_interleaved_char_impl::work(int noutput_items,
                                            gr_vector_const_void_star& input_items,
                                            gr_vector_void_star& output_items)
 {
-    const gr_complex* in = (const gr_complex*)input_items[0];
+    const float* in = (const float*)input_items[0];
     char* out = (char*)output_items[0];
 
-    int npairs = (d_vector ? noutput_items : noutput_items / 2);
-    for (int i = 0; i < npairs; i++) {
-        *out++ = (char)lrintf(in[i].real()); // FIXME saturate?
-        *out++ = (char)lrintf(in[i].imag());
+    for (int i = 0; i < noutput_items; i++) {
```

Comment:
Could potentially move to using Volk's `volk_32f_s32f_convert_8i` here: `void volk_32f_s32f_convert_8i(int8_t* outputVector, const float* inputVector, const float scalar, unsigned int num_points)` ... seems to match up well with this use case! There are versions for `16i` and `32i` as well.